parl/remote/remote_wrapper.py

```diff
@@ -95,13 +101,9 @@ def __init__(self, *args, **kwargs):
         end_of_file = cls_source[1] + len(cls_source[0])
         class_name = cls.__name__
 
-        if '__xparl_proxy_wrapper_nowait__' in kwargs:
-            del kwargs['__xparl_proxy_wrapper_nowait__']
-        if '__xparl_remote_class__' in kwargs:
-            del kwargs['__xparl_remote_class__']
-        if '__xparl_remote_class_max_memory__' in kwargs:
-            del kwargs['__xparl_remote_class_max_memory__']
-
+        for key in list(kwargs.keys()):
+            if key.startswith(XPARL_RESERVED_PREFIX):
+                del kwargs[key]
```

Comment:
print some warning log here.

Reply:
It's not necessary. There are some names starting with `XPARL_RESERVED_PREFIX ` in `kwargs`, which are created by ourselves.

Reply:
ok.